src/components/AutoCompleteSuggestions/index.js

```diff
@@ -1,16 +1,16 @@
 import React from 'react';
-import BaseEmojiSuggestions from './BaseEmojiSuggestions';
+import BaseAutoCompleteSuggestions from './BaseAutoCompleteSuggestions';
 import * as DeviceCapabilities from '../../libs/DeviceCapabilities';
-import {propTypes, defaultProps} from './emojiSuggestionsPropTypes';
+import {propTypes} from './autoCompleteSuggestionsPropTypes';
 
 /**
- * On the mobile-web platform, when long-pressing on emoji suggestions,
- * we need to prevent focus shifting to avoid blurring the main input which makes the emoji picker close and adds the emoji to the composer.
+ * On the mobile-web platform, when long-pressing on auto-complete suggestions,
+ * we need to prevent focus shifting to avoid blurring the main input which makes the suggestions picker close and firing the onSelect callback.
```

Comment:
NAB, suggestion for readability
```suggestion
 * we need to prevent focus shifting to avoid blurring the main input (which makes the suggestions picker close and fires the onSelect callback).
```